databricks/koalas/indexes.py

```diff
@@ -172,9 +172,11 @@ def names(self, names: List[Union[str, Tuple[str, ...]]]) -> None:
             raise ValueError('Names must be a list-like')
         internal = self._kdf._internal
         if len(internal.index_map) != len(names):
+
```

Comment:
I would remove this line. Otherwise looks fine cc @ueshin

Reply:
oops... i should have double checked that.
Thanks for the comment!